dasharo-compatibility/dcu.robot

```diff
@@ -154,7 +154,7 @@ Prepare DCU Test Environment
 
     Download To Host Cache
     ...    logo.bmp
-    ...    https://cloud.3mdeb.com/index.php/s/rsjCdz4wSNesLio/download
+    ...    https://github.com/Dasharo/osfv-test-data/raw/main/coreboot_logo_convert.bmp
     ...    6e5a6722955e4f78d947654630f27ff833703fbc04776ffed963c96617f6bb2a
 
```

Comment:
Does this work?

Shouldn't it be phrased differently? Like:

```
Download To Host Cache
...    coreboot_logo_convert.bmp
...    https://github.com/Dasharo/osfv-test-data/raw/main
...    6e5a6722955e4f78d947654630f27ff833703fbc04776ffed963c96617f6bb2a
```

What is the reason for changing bmp name?

Reply:
that's what i thought when i saw original link that you can find on develop or listed in this issue: https://github.com/Dasharo/open-source-firmware-validation/issues/121#issuecomment-2324689975 
but as it turns out, only the middle part is a link and the string of seemingly random character afterwards is sha256sum argument of Download To Host Cache kwd

Reply:
@JanPrusinowski the link needs the file name too though

Reply:
@wiktormowinski This whole ` Download To Host Cache`  and cache concepts should be removed, at least in places where we have stuff in submodule.

Submodule should be added to this repo as a submodule, and local paths should be used.

Otherwise, there is very little benefit of just changing location from cloud to repo.

Reply:
this actually makes so much sense, idk why didn't I think of that earlier,
working on it: https://github.com/Dasharo/open-source-firmware-validation/pull/485/commits/31ae66be7621ae3e14ee98e1d280a4f900d985b2